e2e/testcafe-devextreme/runner.js

```diff
@@ -54,12 +54,27 @@ createTestCafe({
         setTestingTheme(args);
         setShadowDom(args);
 
+        let sizes = [];
+        switch (componentFolder) {
+            case 'form':
+                sizes = [1200, 1000];
+                break;
+            case 'scheduler':
+                sizes = [1200, 800];
+                break;
+            case 'editors': 
+                sizes = [1200, 800];
+                break;
+            default:
+                sizes = [1200, 800];
+        }
```

Comment:
That's very strange for me. 
At least it can be refactored to reduce copypaste.
But I'm not quire sure it's okay to specify component-specific sized on a runner level